### ISSUE TITLE
Prettier on save output is different from running `format:write` for tailwind classes

### ISSUE THREAD

**Issue by ahkhanjani** [omeralpi/shadcn-phone-input#54]
I only tried for MDX files but the title says everything. This is for this repo itself not the component.

**Comment by omeralpi** [comment]
Hi,
I've noticed that Prettier and VSCode behave differently with MDX formatted files. 
Fixes #55. Could you test this PR? I believe this will resolve the issue.

**Comment by ahkhanjani** [comment]
> I've noticed that Prettier and VSCode behave differently with MDX formatted files.

Hi. I found an [issue](https://github.com/tailwindlabs/prettier-plugin-tailwindcss/issues/244) confirming this.

Running `pnpm format` gave me a different output for both mdx and jsx files. I figured out that [eslint tailwind plugin has different sorting preferences than the prettier plugin](https://github.com/shadcn-ui/taxonomy/issues/73#issuecomment-1549453497). So `lint --fix` was reordering classes that prettier was already fixing with `format` in a different way. Turning off the class order rule fixed the issue for jsx files.

mdx still gives a different output on save. Not sure what we can do about it.

I'll make a PR to this branch with my results.
